### PR TITLE
chore: Remove unused routes

### PR DESCRIPTION
**Pull request by SamoKopecky**
Depends on #159 

SSIA

### CODE REVIEW

**Review by tumido — COMMENTED** [review]
looks good, I have one question though.

File: packages/app/src/App.tsx [hunk 1]
```diff
@@ -6,11 +6,8 @@ import {
   CatalogIndexPage,
   catalogPlugin,
 } from '@backstage/plugin-catalog';
-import {
-  CatalogImportPage,
-  catalogImportPlugin,
-} from '@backstage/plugin-catalog-import';
-import { ScaffolderPage, scaffolderPlugin } from '@backstage/plugin-scaffolder';
+import { catalogImportPlugin } from '@backstage/plugin-catalog-import';
+import { scaffolderPlugin } from '@backstage/plugin-scaffolder';
```
[the remark this answers is not in the document]
Comment (SamoKopecky) [reply packages/app/src/App.tsx]: I removed them, tested it out and everything seems to work fine.

**Review by tumido — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**